### PR TITLE
Update @fortawesome/fontawesome-free

### PR DESCRIPTION
**Pull request by haslinghuis**
Might fix intermittent issue while loading from terminal with `yarn && yarn start` from clean development environment:

image

Thanks to @nerdCopter for spotting we were not using the minified version of `all.css` in main.html

### CODE REVIEW

**Comment by nerdCopter** [comment]
i found the solution!  yes updating is not the solution, but also welcomed.

**Review by nerdCopter — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]

**Pull request merged**
